submit/templates/submit/policy.html

```diff
@@ -14,16 +14,19 @@ <h2 class="is-size-4">Acknowledge policy statement</h2>
   <li>For third-party submissions, I have obtained pre-authorization from arXiv
     to submit as a third-party submitter.</li>
 </ul>
-
+<form method="GET" action="{{ url_for('ui.policy',submission_id=submission_id) }}">
```

Comment:
Should the method be POST?

Reply:
The previous pages are GET...not sure why we chose that after the first one. I suppose if we used POST then we wouldn't need to send the submission_id...right?

Tried POST quickly and current configuration returns a 405. Something to talk about for all pages?

Reply:
Comment on submission_id: for the macro that generates the row of numbered boxes at the top (that will eventually become the progress bar). It doesn't work, because the routes have a submission ID and the links are hard coded. The note about that was to un-hard-code the progress bar links. Apologies for being unclear!